### PR TITLE
BaSP-TG-11: Admin Add and Edit

### PR DESCRIPTION
**Pull request by Baldizan**
Admins list, add and edit modal, and delete option

### CODE REVIEW

**Review by elias-92 — COMMENTED** [review]
reviewed.

**Review by elias-92 — COMMENTED** [review]
revised.

File: src/Components/Admins/modal.module.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    background-color: #fff;
+    border: solid 1px #373867;
+    border-radius: 5px;
+}
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    width: 90%;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.153);
```
Comment (LucianaPrat) [inline src/Components/Admins/modal.module.css:16]: the color in this line should be hex

Comment (martumoser) [reply src/Components/Admins/modal.module.css:16]: not necessary in this case because with rgba color it is possible to add the opacity

**Review by LucianaPrat — APPROVED** [review]

**Review by martumoser — CHANGES_REQUESTED** [review]

**Review by martumoser — APPROVED** [review]

**Review by Martinzancocchia — APPROVED** [review]

**Pull request merged**